src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { useUID } from 'react-uid';
+import styles from './ProgressBar.module.css';
+import Text from '../Text';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Overrides the default "totalValue / max" in cases where a different caption may be desired.
```

Comment:
Should this be changed now that the props have been renamed?
```suggestion
   * Overrides the default "currentValue / maxValue" in cases where a different caption may be desired.
```